README.md

```diff
@@ -1,57 +1,78 @@
-# Project Name
+# How to use Azure SDK for C through Swift
 
-(short, 1-3 sentenced, description of the project)
+This is a sample implementation of the [Embedded C SDK](https://aka.ms/embeddedcsdk) which simplifies the creation of connecting Swift devices to Azure IoT services like Azure Iot Hub and DPS (Device Provisioning Service). 
 
-## Features
+## Architecture view
 
-This project framework provides the following features:
+The illustration below shows the Swift sample taking a dependence on the Embedded C SDK which provides libraries used to access all Azure IoT functionalities. 
 
-* Feature 1
-* Feature 2
-* ...
+This SDK also implements the BYO (bring your own) network stack approach, which means device builders can choose whichever MQTT client, TLS and TCP stack that works better for their target platform. 
```

Comment:
```suggestion
This SDK also implements the BYO (bring your own) network stack approach, which means embedded developers can choose whichever MQTT client, TLS and TCP stack that works better for their target platform.
```

Reply:
My opinion...if it's in Swift on an iOS device, this isn't much of an embedded scenario.